scripts/mirror.js

```diff
@@ -219,74 +280,93 @@ const bumpChromeAndroid = (originalData, sourceData, source) => {
-    } else {
-      if (!chromeFalse && !chromeNull) {
-        if (originalData.version_added == true) {
-          newData.version_added = '≤18';
-        } else {
-          if (
-            sourceData.version_added == true ||
-            Number(sourceData.version_added) <= 79
-          ) {
-            if (originalData.version_added == false) {
-              newData.version_added = '79';
-            } else if (originalData.version_added == null) {
-              newData.version_added = '≤79';
-            }
-          } else {
-            newData.version_added == sourceData.version_added;
-          }
-        }
-      } else if (chromeFalse) {
-        if (originalData.version_added && !originalData.version_removed) {
-          newData.version_removed = '79';
-        }
+/**
+ * @param {SupportStatement} sourceData
+ * @param {SupportStatement} originalData
+ * @returns {SupportStatement}
+ */
+const bumpEdgeFromChrome = (sourceData, originalData) => {
+  let newData = copyStatement(sourceData);
+
+  let chromeFalse =
+    sourceData.version_removed || sourceData.version_added === false;
+  let chromeNull = sourceData.version_added === null;
+
+  if (chromeFalse) {
+    if (originalData.version_added && !originalData.version_removed) {
+      newData.version_removed = '79';
+    }
+  } else if (chromeNull) {
+    newData.version_added = null;
+  } else {
+    if (sourceData.version_added === true) {
+      newData.version_added =
+        originalData.version_added === true ? '≤18' : true;
+    } else if (Number(sourceData.version_added) <= 79) {
+      if (originalData.version_added === false) {
+        newData.version_added = '79';
+      } else if (originalData.version_added === null) {
+        newData.version_added = '≤79';
       }
+    } else {
+      newData.version_added = sourceData.version_added;
     }
+  }
 
-    let newNotes = combineNotes(
-      updateNotes(sourceData.notes, /Chrome/g, 'Edge'),
-      originalData.notes,
-    );
+  let newNotes = updateNotes(sourceData.notes, /Chrome(?! ?OS)/g, 'Edge');
+  if (newNotes) newData.notes = newNotes;
+
+  return newData;
+};
+
+/**
+ * @param {SupportStatement} originalData
+ * @param {SupportStatement} chromeData
+ * @param {SupportStatement} ieData
+ * @returns {SupportStatement}
+ */
+const bumpEdge = (originalData, chromeData, ieData) => {
+  if (Array.isArray(originalData)) {
+    return originalData.map(d => bumpEdge(d, chromeData, ieData));
+  }
 
-    if (newNotes) {
-      newData.notes = newNotes;
+  let newData = [];
+
+  if (ieData) {
+    if (Array.isArray(ieData)) {
+      newData += ieData.map(d => bumpEdgeFromIE(d));
+    } else {
+      newData.push(bumpEdgeFromIE(ieData));
     }
   }
 
-  return newData;
+  if (chromeData) {
+    if (Array.isArray(chromeData)) {
+      newData += chromeData.map(d => bumpEdgeFromChrome(d, originalData));
+    } else {
+      newData.push(bumpEdgeFromChrome(chromeData, originalData));
```

Comment:
If you run `npm run -- mirror edge -m always` an exception will be thrown here: "newData.push is not a function"